src/plugins/intel_gpu/src/graph/network.cpp

```diff
@@ -1150,21 +1150,36 @@ void network::execute_impl(const std::vector<event::ptr>& events) {
 
     set_arguments();
     GPU_DEBUG_GET_INSTANCE(debug_config);
+    int64_t curr_iter = -1;
+#ifdef GPU_DEBUG_CONFIG
+    GPU_DEBUG_IF(debug_config->dump_iteration.empty() != true) {
+        curr_iter = static_cast<int64_t>(iteration);
+        iteration++;
```

Comment:
why do you cast it instead of `curr_iter = iteration++`?

Reply:
The type of iteration was size_t before. It is not now, I will fix it.